docs/en-US/troubleshooting/index.md

```diff
@@ -172,6 +172,16 @@ Delete the folder name that matches the path in error above. In the example abov
 
 Then go back and run `vagrant up --provision` again.
 
+
+### WP CLI Packages provisioner errors
+
+If the tools provisioner errors out while installing or updating WP CLI packages, this may be related to a package problem within WP CLI.
+
+You may use `vagrant ssh` and `wp package` to address the problem based on the error logs produced by the provisioner.
```

Comment:
Just specify what is the right folder (the root folder) to avoid confusions.

Reply:
https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org/pull/208/commits/22bfd003c1f1779521a3b0717da689cda04c05ab this ok?